pkgs/os-specific/linux/evdi/default.nix

```diff
@@ -2,25 +2,25 @@
 
 stdenv.mkDerivation rec {
   pname = "evdi";
-  version = "-unstable-20190116";
+  version = "-unstable-20200222";
```

Comment:
This should just be `unstable-20200222` and not `-unstable-20200222`.

Reply:
Fixed.